devices/sinope.js

```diff
@@ -82,26 +87,46 @@ module.exports = [
         vendor: 'Sinopé',
         description: 'Zigbee line volt thermostat',
         meta: {thermostat: {dontMapPIHeatingDemand: true}},
-        fromZigbee: [
-            fz.legacy.sinope_thermostat_att_report, fz.legacy.hvac_user_interface, fz.electrical_measurement, fz.metering,
+        fromZigbee: [fz.legacy.sinope_thermostat_att_report, fz.legacy.hvac_user_interface, fz.electrical_measurement, fz.metering,
             fz.ignore_temperature_report, fz.legacy.sinope_thermostat_state, fz.sinope_thermostat],
-        toZigbee: [
-            tz.thermostat_local_temperature, tz.thermostat_occupied_heating_setpoint, tz.thermostat_unoccupied_heating_setpoint,
+
+        toZigbee: [tz.thermostat_local_temperature, tz.thermostat_occupied_heating_setpoint, tz.thermostat_unoccupied_heating_setpoint,
             tz.thermostat_temperature_display_mode, tz.thermostat_keypad_lockout, tz.thermostat_system_mode, tz.thermostat_running_state,
-            tz.sinope_thermostat_occupancy, tz.sinope_thermostat_backlight_autodim_param, tz.sinope_thermostat_time,
-            tz.sinope_thermostat_enable_outdoor_temperature, tz.sinope_thermostat_outdoor_temperature, tz.sinope_time_format],
+            tz.sinope_thermostat_backlight_autodim_param, tz.sinope_thermostat_time, tz.sinope_thermostat_enable_outdoor_temperature,
+            tz.sinope_thermostat_outdoor_temperature, tz.thermostat_pi_heating_demand, tz.sinope_thermostat_occupancy,
+            tz.electrical_measurement_power, tz.currentsummdelivered, tz.acvoltage, tz.accurrent],
         exposes: [
             exposes.climate()
+                .withLocalTemperature()
                 .withSetpoint('occupied_heating_setpoint', 5, 30, 0.5)
                 .withSetpoint('unoccupied_heating_setpoint', 5, 30, 0.5)
-                .withLocalTemperature()
-                .withSystemMode(['off', 'heat']).withRunningState(['idle', 'heat'])
-                .withPiHeatingDemand(),
-            exposes.enum('thermostat_occupancy', ea.SET, ['unoccupied', 'occupied'])
+                .withSystemMode(['off', 'heat'], ea.ALL, 'Mode of the thermostat')
+                .withRunningState(['idle', 'heat'])
+                .withPiHeatingDemand(ea.STATE_GET),
+            exposes.enum('thermostat_occupancy', ea.ALL, ['unoccupied', 'occupied'])
                 .withDescription('Occupancy state of the thermostat'),
-            exposes.enum('backlight_auto_dim', ea.SET, ['on demand', 'sensing'])
+            exposes.enum('backlight_auto_dim', ea.ALL, ['on demand', 'sensing'])
                 .withDescription('Control backlight dimming behavior'),
-            e.keypad_lockout(), e.energy(), e.power(), e.current(), e.voltage()],
+            exposes.enum('keypad_lockout', ea.ALL, ['unlock', 'lock1'])
+                .withDescription('Enables or disables the device’s buttons'),
+            exposes.numeric('power', ea.STATE_GET)
```

Comment:
Instead of this use just `e.power()` (same for current/voltage/energy)

Reply:
yes but can e.power()  acess be set to STATE_GET?

Reply:
`e.power().withAccess(ea.STATE_GET)`

Reply:
Done, had to set ti for all. Otherwise i get complaints